docs/user/quick-start-browser.md

```diff
@@ -0,0 +1,270 @@
+import { initializeAgent, getDefaultInstrumentations } from '@grafana/agent-web';
+
+const agent = initializeAgent({
+  url: 'http://localhost:12345/collect',
+  apiKey: 'secret',
+  instrumentations: [...getDefaultInstrumentations(), new TracingInstrumentation()],
+  app: {
+    name: 'frontend',
+    version: '1.0.0',
+  },
+});
+
+// get otel trace and context apis
+const { trace, context } = agent.api.getOTEL();
+
+const tracer = trace.getTracer('default');
+const span = tracer.startSpan('click');
+context.with(trace.setSpan(context.active(), span), () => {
+  doSoemthing();
+  span.end();
+})
+```
+
+### With custom Open Telemetry tracing configuration
+
+Configure OTEL manually. Use `GrafanaAgentTraceExporter` and call `agent.api.initOTEL`
+with OTEL trace and context APIs.
+
+```javascript
+import { trace, context } from '@opentelemetry/api';
+import { ZoneContextManager } from '@opentelemetry/context-zone';
+import { W3CTraceContextPropagator } from '@opentelemetry/core';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+import { DocumentLoadInstrumentation } from '@opentelemetry/instrumentation-document-load';
+import { FetchInstrumentation } from '@opentelemetry/instrumentation-fetch';
+import { UserInteractionInstrumentation } from '@opentelemetry/instrumentation-user-interaction';
+import { XMLHttpRequestInstrumentation } from '@opentelemetry/instrumentation-xml-http-request';
+import { Resource } from '@opentelemetry/resources';
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base';
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+import { initializeAgent } from '@grafana/agent-web';
+import { GrafanaAgentTraceExporter } from '@grafana/agent-trace-web';
+
+const VERSION = '1.0.0';
+const NAME = 'frontend';
+const COLLECTOR_URL = 'http://localhost:12345/collect';
+
+// initialize agent
+const agent = initializeAgent({
+  url: COLLECTOR_URL,
+  apiKey: 'secret',
+  app: {
+    name: NAME,
+    version: VERSION,
+  },
+});
+
+// set up otel
+const resource = Resource.default().merge(new Resource({
+  [SemanticResourceAttributes.SERVICE_NAME]: NAME,
+  [SemanticResourceAttributes.SERVICE_VERSION]: VERSION
+}));
+
+const provider = new WebTracerProvider({ resource });
+provider.addSpanProcessor(new BatchSpanProcessor(new GrafanaAgentTraceExporter({ agent })));
+provider.register({
+  propagator: new W3CTraceContextPropagator(),
+  contextManager: new ZoneContextManager(),
+});
+const ignoreUrls = [COLLECTOR_URL];
+registerInstrumentations({
+  instrumentations: [
+    new DocumentLoadInstrumentation() as any,
+    new FetchInstrumentation({ ignoreUrls }),
+    new XMLHttpRequestInstrumentation({ ignoreUrls }),
+    new UserInteractionInstrumentation()
+  ]
+});
+
+// register otel with agent
+agent.api.initOTEL(trace, context);
+```
+
+## Usage
```

Comment:
```suggestion
## Use
```

Reply:
Usually we try to use "use' but not sure this isn't a specific use case, so do correct if wrong.